forum/qa-include/pages/question-view.php

```diff
@@ -697,11 +697,13 @@ function qa_page_q_comment_follow_list($question, $parent, $commentsfollows, $al
 				$showcomments[$commentfollowid]=$commentfollow;
 
 		$countshowcomments=count($showcomments);
-
-		if ( (!$alwaysfull) && ($countshowcomments > qa_opt('show_fewer_cs_from')) )
+		/* Show all comments */
+		$skipfirst = 0;
+		/* ----------------- */

```

Comment:
Komentarze napisałem po to, aby można było łatwo zobaczyć, do czego jaka zmiana jest, bez patrzenia w repo.